guides/release/testing/unit-testing-basics.md

```diff
@@ -138,14 +139,15 @@ Suppose we have an object that has a property and a method observing that proper
 import Service from '@ember/service';
 import { observer } from "@ember/object";
 
-export default Service.extend({
-  foo: 'bar',
-  other: 'no',
+export default class SomeThingService extends Service {
```

Comment:
Lets revert this change for now, I don't think we can actually talk about observers in the new world (because there is no decorator form of `observes` in core, and ultimately folks should _probably_ move away from them).